compile.go

```diff
@@ -704,90 +641,86 @@ func ToQueryTime(value values.Value) (Time, error) {
 }
 
 type importer struct {
+	r    *runtime
 	pkgs map[string]*interpreter.Package
 }
 
 func (imp *importer) Copy() *importer {
-	packages := make(map[string]*interpreter.Package, len(imp.pkgs))
-	for k, v := range imp.pkgs {
-		packages[k] = v.Copy()
-	}
-	return &importer{
-		pkgs: packages,
-	}
+	return &importer{r: imp.r}
```

Comment:
Why do we no longer copy packages for an importer? Should we just remove the `Copy` method entirely from the `importer`?

Reply:
We don't. It seems the function is now unused so I removed it. The original rationale is that we want to copy the runtime, but we want to reevaluate the packages when a copy happens. But the method is unused so there's no need to explain why it doesn't copy packages.